TaskfMRIAnalysis/scripts/TaskfMRILevel1.sh

```diff
@@ -374,7 +383,11 @@ if [[ $runParcellated == true || $runDense == true ]]; then
 	rm ${ResultsFolder}/${LevelOnefMRIName}/${LevelOnefMRIName}_Atlas${SmoothingString}${RegString}${ParcellationString}_FAKENIFTI.nii.gz ${ResultsFolder}/${LevelOnefMRIName}/${LevelOnefMRIName}_Atlas${SmoothingString}${RegString}${ParcellationString}_FAKENIFTI_mean.nii.gz
 fi
 
-if $runVolume; then
+if [[ "${TemporalFilter}" == "-1" && $runVolume == true]]; then
+	# We drop the "dilMrim" string from the output file name, so as to avoid breaking
```

Comment:
What is dilMrim?

Reply:
The structure of this seems a little odd, equivalent to:
```
if B and runVolume
...
else
    if runVolume
...
    fi
fi
```
I would have done this:
```
if runVolume
    if B
...
    else
...
    fi
fi
```
Persinally, I avoid elif, as mostly being a construct to substitute for case/switch (which in some languages can't handle strings), but bash case handles strings (and even globbing).

Reply:
> What is dilMrim?

'dilMrim' is the string in the filename of the volume image after edge dilation. (The volume dilation has been around for ages... There's a long explanation starting at the line '#Add volume dilation'.)

In the current code, the output filename of the fslmaths -bptf call removes the 'dilMrim' string. Therefore, if we skip temporal filtering, we still need to remove that 'dilMrim' string.

(The fact that this string is added in volume smoothing and then removed in temporal smoothing needs to be fixed so that we can skip volume smoothing and temporal smoothing and still end up with the right filename. I'll need to address this at some point in the future.)

Reply:
I added that code a long time ago.  Might just be easiest (and less prone to error) to leave it as is, and just have a conditional that checks for absence of temporal smoothing and then renames as necessary.